src/applications/disability-benefits/all-claims/sass/disability-benefits.scss

```diff
@@ -12,3 +12,11 @@
 .full-page-alert {
   margin-bottom: 2em;
 }
+
+button {
```

Comment:
I think this is probably too broad a rule to put on the whole form. Is this a temporary fix?

Reply:
Yeah, this will need to change, but might be a good idea to make this more explicit.